CSharp/ZgTimeTable/ZgTimeTable/TimeTable.cs

```diff
@@ -153,5 +179,9 @@ public long remaining(long time)
             long r = e - t;
             return r > 0 ? r : 0;
         }
+        public bool isInPeriod(long time)
+        {
+            return time >= Start && time < End;
```

Comment:
Second commit has not taken into account. this method and some other things should change